module/VuFind/sql/pgsql.sql

```diff
@@ -189,6 +189,22 @@ UNIQUE (session_id)
 CREATE INDEX last_used_idx on session(last_used);
 
 --
+-- Table structure for table external_session
+--
+
+DROP TABLE IF EXISTS "external_session";
+
+CREATE TABLE external_session (
+id SERIAL,
+session_id varchar(128),
```

Comment:
I just double-checked, and there doesn't seem to be an inconsistency in the session table:

MySQL: `session_id` varchar(128) DEFAULT NULL,

PostgreSQL: session_id varchar(128),

Both of these allow, and default to, NULL. Am I missing something somewhere else?